src/store/models/designer.spec.ts

```diff
@@ -309,6 +311,7 @@ describe('Designer model', () => {
       beforeEach(() => {
         const { setActiveId } = store.getActions().designer;
         setActiveId(firstNetwork().id);
+        mockDetectPort.mockImplementation(port => Promise.resolve(port));
```

Comment:
I'm not following why you need to change the mock's implementation in this tests. It will succeed without this.

Reply:
Removed.